app/components/text-input/text-input.tsx

```diff
@@ -7,9 +7,11 @@ const TextInput = ({label, id, className, error, ...props}: TextInputProps) => {
   return (
     <div className="flex flex-col">
       <div className="flex justify-between mb-2.5">
-        <label htmlFor={id} className={`text-xs font-bold leading-4 ${error ? 'text-danger' : ''}`}>
-          { label }
-        </label>
+        { label && (
+          <label htmlFor={id} className={`text-xs font-bold leading-4 ${error ? 'text-danger' : ''}`}>
+            { label }
+          </label>
+        )}
```

Comment:
This just modifies the base TextInput component to be able to render inputs that don't have labels